content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, ClickHouse, ClickHouse Operator, ClickHouse Cluster'
+description: 'Learn how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.'
+linkTitle: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+weight: 14340
+---
+
+[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). ClickHouse on QingCloud is a ClickHouse cluster application deeply customized with the following features.
+
+- Automated cluster management. ClickHouse on QingCloud manages cluster status via Paxos protocols. As you run ClickHouse on QingCloud and manage it like a single-node ClickHouse cluster, you will be free from complicated cluster management operations.
```

Comment:
ClickHouse on QingCloud manages cluster status via Paxos protocols.
->
ClickHouse on QingCloud manages cluster status via the Paxos protocol.